src/sys/mod.rs

```diff
@@ -226,3 +226,7 @@ feature! {
     #![feature = "time"]
     pub mod timer;
 }
+
+#[cfg(all(target_os = "linux", feature = "signal", feature = "process"))]
+/// pidfd related functionality
```

Comment:
This mod is public. I think it's better to have a capitalized description inside `src/sys/pidfd.rs` instead, to align with other mods like `memfd`.

Reply:
Sounds good, fixed.